src/libs/actions/QueuedOnyxUpdates.ts

```diff
@@ -25,6 +25,8 @@ function queueOnyxUpdates(updates: OnyxUpdate[]): Promise<void> {
 }
 
 function flushQueue(): Promise<void> {
+    let copyUpdates = [...queuedOnyxUpdates];
+    queuedOnyxUpdates = [];
```

Comment:
Can you add a comment explaining that this is needed to prevent a race condition?